src/main/java/com/cloudbees/jenkins/support/BundleBrowser.java

```diff
@@ -0,0 +1,108 @@
+package com.cloudbees.jenkins.support;
```

Comment:
@aheritier @christ66 Is it fine for GSoC? I have some doubts about it, so maybe creating a community package would be preferable